content/matplotlib/concepts/pyplot/terms/pie/pie.md

```diff
@@ -0,0 +1,119 @@
+
+`.pie()` takes the following arguments:
+
+-  `x`: 1D array-like object that specifies the wedge sizes.
+
+-  `explode`: Array-like object which specifies the fraction of the radius with which to offset each wedge.
+
+-  `labels`: List of strings providing the labels for each wedge.
+
+-  `colors`: Array-like sequence of colors through which the pie chart will cycle.
+
+-  `autopct`: A string or function used to label the wedges with their numeric value. The label will be placed inside the wedge.
+
+-  `pctdistance`: Float that sets the relative distance along the radius at which the the text generated by _autopct_ is drawn.
+
+-  `shadow`: Boolean value, draws a shadow beneath the pie.
+
+-  `labeldistance`: Float that sets the relative distance along the radius at which the labels are drawn.
+
+-  `startangle`: Float that sets the angle by which the start of the pie is rotated, counterclockwise from the x-axis.
+
+-  `radius`: Float that sets the radius of the pie.
+
+-  `counterclock`: Boolean to specify fractions direction, clockwise or counterclockwise.
+
+-  `wedgeprops`: Dict of arguments passed to each `patches.Wedge`
+
+-  `textprops`: Dict of arguments to pass to the text objects.
+
+-  `center`: (float, float), the coordinates of the center of the chart.
+
+-  `frame`: Bool, plot Axes frame with the chart if true.
+
+-  `rotatelabels`: Rotate each label to the angle of the corresponding slice if true.
+
+-  `normalize`: Bool, When _True_, always make a full pie by normalizing x. _False_makes a partial pie.
+
+-  `hatch`: Str or list, Hatching pattern applied to all pie wedges or sequence of patterns through which the chart will cycle.
+
+  ## Examples
+
+ 
+Examples below demonstrate the use of `.pie()` to plot values and show the functionality of some arguments.
+
+A simple pie chart: 
+
+```py
+
+# Import libraries
+from matplotlib import pyplot as plt
+
+# Creating dataset
+pizza = ['CHEESE','PEPPERONI','VEGGIE','MEAT','BBQ CHICKEN','HAWAIIAN']
+data = [23,  17,  35,  29,  12,  41]
+
+# Creating plot
+plt.pie(data, labels = pizza)
+
+# show plot
+plt.show()
+
+```
+
+Output:
+
+![Output of matplotlib.pyplot.pie() function example 1](/media/matplotlib-pie-1.png)
+
+
+Customizing the pie chart:
+
+```py
+
+# Import libraries
+import matplotlib.pyplot as plt
+
+# Plot data
+pizza = ['CHEESE','PEPPERONI','VEGGIE','MEAT','BBQ CHICKEN','HAWAIIAN']
+popularity = [22.2,  17.6,  8.8,  8,  7.7,  6.7]
+
+#colors = ['red', 'green', 'yellow', 'blue', 'purple', 'brown']
+colors = ['#FF6188',  '#A9DC76',  '#FFD866',  '#78DCE8',  '#AB9DF2',  '#8c564b']
+
+# explode 1st slice
+explode = (0.1,  0,  0,  0,  0,  0)
+
+# Plot
+plt.pie(popularity, explode=explode, labels=pizza, colors=colors, autopct='%1.1f%%', shadow=True, startangle=140)
+plt.show()
+
+```
+
+Output:
+
+![Output of matplotlib.pyplot.pie() function example 2](/media/matplotlib-pie-2.png)
+
```

Comment:
please update images after making the changes above and delete images no longer in use.